modules/nf-core/thermorawfileparser/main.nf

```diff
@@ -0,0 +1,65 @@
+process THERMORAWFILEPARSER {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/thermorawfileparser:1.4.3--ha8f3691_0' :
+        'biocontainers/thermorawfileparser:1.4.3--ha8f3691_0' }"
+
+    input:
+    tuple val(meta), path(raw)
+
+    output:
+    tuple val(meta), path("*.mzML")      , optional:true , emit: mzml
+    tuple val(meta), path("*.mzML.gz")   , optional:true , emit: mzml_gz
+    tuple val(meta), path("*.mgf")       , optional:true , emit: mgf
+    tuple val(meta), path("*.mgf.gz")    , optional:true , emit: mgf_gz
+    tuple val(meta), path("*.parquet")   , optional:true , emit: parquet
+    tuple val(meta), path("*.parquet.gz"), optional:true , emit: parquet_gz
```

Comment:
I'm not a big fan of this, as this would make this module difficult to make it compatible in a pipeline setting. 
```suggestion
    tuple val(meta), path("*.{mzML,mzML.gz,mgf,mgf.gz,parquet,parquet.gz}")      , emit: spectra
```